### PR TITLE
Add TRE_URL to .env.example file

### PR DESCRIPTION
**Pull request by martinpeck**
Previous PR (#1441) added additional environment variables into the mix for `make bundle-register`

The result is that without the `TRE_URL` env var set you may see an error when you run `make bundle-register` in the situation where you have these two defined env vars...

```
AUTOMATION_ADMIN_ACCOUNT_CLIENT_ID
AUTOMATION_ADMIN_ACCOUNT_CLIENT_SECRET
```

You may see this error:

image

This PR adds an addition environment variable into the `.env.example` file that the user needs to copy to the .env file within the `templates/core/` folder.

THIS IS FAR FROM PERFECT. We absolutely should create this env var ourselves, but after reviewing this for a while I realised that the .env file set up in the project is a bit messy (tracked in #1470)

For now, consider this PR as some sign-posting/documentation.

### CODE REVIEW

File: templates/core/.env.sample [hunk 1]
```diff
@@ -15,3 +15,7 @@ SWAGGER_UI_CLIENT_ID=__CHANGE_ME__
 # Automation accounts (e.g. bundle registration)
 AUTOMATION_ADMIN_ACCOUNT_CLIENT_ID=__CHANGE_ME__
 AUTOMATION_ADMIN_ACCOUNT_CLIENT_SECRET=__CHANGE_ME__
+
+# Bundle Registration via Automation needs to know TRE_URL.
```
Comment (marrobi) [inline templates/core/.env.sample:19]: Thinking....

Can we put these "optional" or rather CI/CD type variables under a separate header saying "Automated deployments such as deployments which are part of CI/CD pipelines will need the following variables:"

For users wanting to kick the tires, they do not need to set these, or the above automation env vars, so is slightly confusing.

Happy to discuss.

Comment (martinpeck) [reply templates/core/.env.sample:19]: so, comment all three of these out in the example?

Comment (marrobi) [reply templates/core/.env.sample]: Great. Thanks.

**Review by marrobi — APPROVED** [review]
LGTM

**Pull request merged**